tests/src/unit/tests/test_connection.cpp

```diff
@@ -35,14 +35,25 @@ using namespace datastax::internal::core;
 
 namespace {
 
-static void setenv(const std::string& name, const std::string& value) {
+void setenv(const std::string& name, const std::string& value) {
 #ifdef _WIN32
   _putenv(const_cast<char*>(std::string(name + "=" + value).c_str()));
 #else
   ::setenv(name.c_str(), value.c_str(), 1);
 #endif
 }
 
+String current_dir()
+{
+  char buffer[256];
+#ifdef _WIN32
+  _getcwd(buffer, 256);
+#else
+  getcwd(buffer, 256);
```

Comment:
Should ideally use sizeof(buffer) for both of these later references, to avoid having to replicate a constant.
```suggestion
  _getcwd(buffer, sizeof(buffer));
#else
  getcwd(buffer, sizeof(buffer));
```